<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1231</width>
    <height>770</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Estado de Veh칤culos</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>img/Icono.jpg</normaloff>img/Icono.jpg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{
background-color:#f4eee2
}
    QPushButton {
        background-color: #d6cfc3; /* Nuevo color de fondo del bot칩n */
        border: 2px solid #b0a99f; /* Color y grosor del borde */
        border-radius: 15px; /* Bordes redondeados */
        padding: 10px; /* Espaciado interno */
        font-size: 16px; /* Tama침o de la fuente */
        color: #333; /* Color del texto */
        text-align: center; /* Centrar texto */
    }

    QPushButton:hover {
        background-color: #c1b1a4; /* Nuevo color de fondo al pasar el mouse */
    }

    QLineEdit {
        background-color: #ffffff; /* Color de fondo del QLineEdit */
        border: 2px solid #b0a99f; /* Borde del QLineEdit */
        border-radius: 10px; /* Bordes redondeados */
        padding: 8px; /* Espaciado interno */
        font-size: 14px; /* Tama침o de la fuente */
        color: #333; /* Color del texto */
    }

    QComboBox {
             background-color: #d6cfc3; /* Nuevo color de fondo del bot칩n */
        border: 2px solid #b0a99f; /* Color y grosor del borde */
        border-radius: 15px; /* Bordes redondeados */
        padding: 10px; /* Espaciado interno */
        font-size: 16px; /* Tama침o de la fuente */
        color: #333; /* Color del texto */
        text-align: center; /* Centrar texto */
    }</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0" colspan="6">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>Unispace</family>
        <pointsize>50</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: transparent;
color: black</string>
      </property>
      <property name="text">
       <string>Estado de veh칤culos</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QLabel" name="label_7">
      <property name="font">
       <font>
        <family>Liberation Serif</family>
        <pointsize>20</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: transparent;
color: black</string>
      </property>
      <property name="text">
       <string>Buscar</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLineEdit" name="txtBuscarVehiculo">
      <property name="styleSheet">
       <string notr="true">    QLineEdit {
        background-color: #ffffff;
        border: 2px solid #b0a99f;
        border-radius: 10px;
        padding: 8px;
    }</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>Ingresa Dominio o Veh칤culo...</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QPushButton" name="btnBuscarEstado">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: transparent;
border-radius: 25px;</string>
      </property>
      <property name="text">
       <string>游댌</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="4">
     <widget class="QTableWidget" name="tblVehiculos">
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <property name="columnCount">
       <number>5</number>
      </property>
      <property name="horizontalHeaderLabels" stdset="0">
       <stringlist>
        <string>Patente</string>
        <string>Marca</string>
        <string>Modelo</string>
        <string>Color</string>
        <string>Carrocer칤a</string>
        <string>Combustible</string>
       </stringlist>
      </property>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string>ID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Dominio</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Marca</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Modelo</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Color</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="2" column="4" colspan="2">
     <widget class="QTableWidget" name="tblEstVehiculos">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <property name="columnCount">
       <number>8</number>
      </property>
      <property name="horizontalHeaderLabels" stdset="0">
       <stringlist>
        <string>Patente</string>
        <string>Marca</string>
        <string>Modelo</string>
        <string>Color</string>
        <string>Carrocer칤a</string>
        <string>Combustible</string>
       </stringlist>
      </property>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string>Dominio</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>ID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>ID Veh칤culo</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Estado general</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Deudas</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Papeles</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Kilometraje</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Fecha service</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_4">
      <property name="font">
       <font>
        <family>Liberation Serif</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Dominio</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1" colspan="2">
     <widget class="QLineEdit" name="txtDominio">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">    QLineEdit {
        background-color: #ffffff;
        border: 2px solid #b0a99f;
        border-radius: 10px;
        padding: 8px;
    }</string>
      </property>
     </widget>
    </item>
    <item row="3" column="4">
     <widget class="QLabel" name="label_6">
      <property name="font">
       <font>
        <family>Liberation Serif</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Estado general</string>
      </property>
     </widget>
    </item>
    <item row="3" column="5">
     <widget class="QLineEdit" name="txtEstadoGral">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>240</width>
        <height>0</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">    QLineEdit {
        background-color: #ffffff;
        border: 2px solid #b0a99f;
        border-radius: 10px;
        padding: 8px;
    }</string>
      </property>
      <property name="placeholderText">
       <string/>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_5">
      <property name="font">
       <font>
        <family>Liberation Serif</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Marca</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1" colspan="2">
     <widget class="QLineEdit" name="txtMarca">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">    QLineEdit {
        background-color: #ffffff;
        border: 2px solid #b0a99f;
        border-radius: 10px;
        padding: 8px;
    }</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="4" column="4">
     <widget class="QLabel" name="label_8">
      <property name="font">
       <font>
        <family>Liberation Serif</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Tiene deudas?</string>
      </property>
     </widget>
    </item>
    <item row="4" column="5">
     <widget class="QComboBox" name="cmbDeuda">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <item>
       <property name="text">
        <string>No</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Si</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <family>Liberation Serif</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Modelo</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1" colspan="2">
     <widget class="QLineEdit" name="txtModelo">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">    QLineEdit {
        background-color: #ffffff;
        border: 2px solid #b0a99f;
        border-radius: 10px;
        padding: 8px;
    }</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="5" column="4">
     <widget class="QLabel" name="label_9">
      <property name="font">
       <font>
        <family>Liberation Serif</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Tiene papeles?</string>
      </property>
     </widget>
    </item>
    <item row="5" column="5">
     <widget class="QComboBox" name="cmbPapeles">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <item>
       <property name="text">
        <string>Si</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>No</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_12">
      <property name="font">
       <font>
        <family>Liberation Serif</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Color</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1" colspan="2">
     <widget class="QLineEdit" name="txtColor">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">    QLineEdit {
        background-color: #ffffff;
        border: 2px solid #b0a99f;
        border-radius: 10px;
        padding: 8px;
    }</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="6" column="4">
     <widget class="QLabel" name="label_10">
      <property name="font">
       <font>
        <family>Liberation Serif</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Kilometraje</string>
      </property>
     </widget>
    </item>
    <item row="6" column="5">
     <widget class="QLineEdit" name="txtKilometraje">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>240</width>
        <height>0</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">    QLineEdit {
        background-color: #ffffff;
        border: 2px solid #b0a99f;
        border-radius: 10px;
        padding: 8px;
    }</string>
      </property>
      <property name="placeholderText">
       <string/>
      </property>
     </widget>
    </item>
    <item row="7" column="4">
     <widget class="QLabel" name="label_11">
      <property name="font">
       <font>
        <family>Liberation Serif</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Fecha ultimo service</string>
      </property>
     </widget>
    </item>
    <item row="7" column="5">
     <widget class="QDateEdit" name="dtpFechaService">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QDateEdit {
        background-color: #ffffff;
        border: 2px solid #b0a99f;
		border-top-left-radius: 10px;
		border-top-right-radius: 0px;
		border-bottom-left-radius: 10px;
		border-bottom-right-radius: 0px;
        padding: 5px;
    }
</string>
      </property>
      <property name="displayFormat">
       <string>dd/MM/yyyy</string>
      </property>
     </widget>
    </item>
    <item row="9" column="0" colspan="3">
     <widget class="QLineEdit" name="txtID">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">    QLineEdit {
        background-color: #ffffff;
        border: 2px solid #b0a99f;
        border-radius: 10px;
        padding: 8px;
    }</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="8" column="5" alignment="Qt::AlignRight">
     <widget class="QPushButton" name="btnGuardar">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">    QPushButton {
        background-color: #d6cfc3;
        border: 2px solid #b0a99f;
        border-radius: 10px;
        padding: 10px;
    }

    QPushButton:hover {
        background-color: #c1b1a4;
    }
</string>
      </property>
      <property name="text">
       <string>Guardar estado</string>
      </property>
     </widget>
    </item>
    <item row="8" column="2" alignment="Qt::AlignRight">
     <widget class="QPushButton" name="btnEditarEstado">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">    QPushButton {
        background-color: #d6cfc3;
        border: 2px solid #b0a99f;
        border-radius: 10px;
        padding: 10px;
    }

    QPushButton:hover {
        background-color: #c1b1a4;
    }
</string>
      </property>
      <property name="text">
       <string>Editar estado</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
